### PR TITLE
Export attendees and entries

### PR DESCRIPTION
**Pull request by tsource98**
This PR adds:

- Task to export attendees and entries

### CODE REVIEW

File: lib/mix/tasks/export.attendees.entries.ex [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+    IO.puts("uuid,name,email,entries")
+
+    Enum.each(
+      Safira.Accounts.list_active_attendees(),
+      fn a ->
+        IO.puts(csv_io(a))
+      end
+    )
+  end
```
Comment (joaop21) [inline lib/mix/tasks/export.attendees.entries.ex:15]: I think you could just:
```Elixir
Safira.Accounts.list_active_attendees()
|> Enum.each(fn a -> csv_io(a) |> Mix.shell().info end)
```

**Review by joaop21 — CHANGES_REQUESTED** [review]
Don't use `IO.puts` in tasks, instead use `Mix.shell().info`.

File: lib/mix/tasks/export.attendees.entries.ex (hunk 1 above)
Comment (tsource98) [reply lib/mix/tasks/export.attendees.entries.ex]: I understand your tip, nice one. Thx 👍
About IO.puts vs Mix.shell().info, I just tried to make that code consistent when compared to other tasks to export information(e.g., export.attendees.ex). Anyway, mix.shell().info has been implemented on this task

**Comment by jpcorreia99** [comment]
Isn't there any need to have the output sorted by entries or the output in a file?

File: lib/mix/tasks/export.attendees.entries.ex [hunk 2]
```diff
@@ -0,0 +1,18 @@
+defmodule Mix.Tasks.Export.Attendees.Entries do
+  use Mix.Task
+
+  def run(_) do
+    Mix.Task.run("app.start")
+
+    IO.puts("uuid,name,email,entries")
```
Comment (joaop21) [inline lib/mix/tasks/export.attendees.entries.ex:7]: IO.puts -> Mix.shell().info

**Review by joaop21 — CHANGES_REQUESTED** [review]

**Review by joaop21 — APPROVED** [review]
Assuming everything is well tested 🚀

**Review by jpcorreia99 — APPROVED** [review]

**Pull request merged**